lldb/test/API/lang/swift/stdlib/ContiguousArray/TestContiguousArray.py

```diff
@@ -0,0 +1,39 @@
+# TestContiguousArray.py
+#
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2014 - 2016 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See https://swift.org/LICENSE.txt for license information
+# See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+#
+# ------------------------------------------------------------------------------
```

Comment:
I don't think we need to slap a license header into tests, if we do, we should update the copyright, but I would just remove it.